schema/pgsql/upgrades/026.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TYPE incident_history_event_type ADD VALUE 'muted' AFTER 'opened';
+ALTER TYPE incident_history_event_type ADD VALUE 'unmuted' AFTER 'muted';
+ALTER TYPE notification_state_type ADD VALUE 'suppressed' BEFORE 'pending';
+
+ALTER TABLE object
+    ADD COLUMN is_muted boolenum NOT NULL DEFAULT 'n',
```

Comment:
This leaves behind a default value for the column which is inconsistent with the full schema file.